src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -328,7 +333,7 @@ exports.createFiberFromText = function(content : string, priorityLevel : Priorit
   return fiber;
 };
 
-function createFiberFromElementType(type : mixed, key : null | string) : Fiber {
+function createFiberFromElementType(type : mixed, key : null | string, debugOwner : null | Fiber | ReactInstance) : Fiber {
```

Comment:
I think that makes sense. The two ideas i was going to explore was to either create something like `getDebugFiberWorkInProgress` or floundering my way through something like what you did.

What were the other solutions you tried that didn’t work out?